Sources/Common/EventMapping.swift

```diff
@@ -30,7 +30,20 @@ open class EventMapping<Event> {
         eventMapper = mapping
     }
 
+    @available(*, renamed: "fire(_:error:parameters:)")
     public func fire(_ event: Event, error: Error? = nil, parameters: [String: String]? = nil, onComplete: @escaping (Error?) -> Void = {_ in }) {
         eventMapper(event, error, parameters, onComplete)
     }
+
+    public func asyncFire(_ event: Event, error: Error? = nil, parameters: [String: String]? = nil) async throws {
```

Comment:
This method seems to blocks forever in macOS (and I believe in iOS too) due to a bug that's in the clients.  The bug went unnoticed because previously we weren't relying on this completion closure.

The completion closure is being ignored in the clients [here](https://github.com/duckduckgo/macos-browser/blob/b6e6691eaf04dd6eebfb636a1f0dff11eb1436a7/DuckDuckGo/NetworkProtection/NetworkExtensionTargets/NetworkExtensionTargets/MacPacketTunnelProvider.swift#L137) and [here](https://github.com/duckduckgo/iOS/blob/5f89b4be78522cd0d801be35defc7815b333cef1/PacketTunnelProvider/NetworkProtection/NetworkProtectionPacketTunnelProvider.swift#L47).